modules/ocf_apphost/manifests/init.pp

```diff
@@ -9,8 +9,17 @@
     cron => true;
   }
 
-  # useful for supervising user processes
-  package { 'daemontools':; }
+  # enable a persistent per-user systemd for all ocfdev users
+  file { '/var/lib/systemd/linger':
+    ensure  => directory,
+    # This makes sure only ocfdev users get per-user systemd. TODO: is this
```

Comment:
I think that only allowing ocfdev users to get per-user systemd on the apphost is reasonable. What do others think?